tests/data/metadata_good/foo_webassets_v1.json

```diff
@@ -0,0 +1,16 @@
+{
+    "images": [
+        {
+            "id": "foowebassets123"
+        }
+    ],
+    "repository": "foo-webassets",
+    "repo-registry-id": "foo/webassets",
```

Comment:
This doesn't look like the sort of repository or repo-registry-id we'd expect to see. Why is 'webassets' there?